pkg/controller/stacks/install/installjob.go

```diff
@@ -40,6 +40,13 @@ import (
 	"github.com/crossplaneio/crossplane/pkg/stacks"
 )
 
+// Labels used to track ownership across namespaces and scopes.
+const (
+	labelInstallNamespace = "stacks.crossplane.io/install-namespace"
+	labelInstallName      = "stacks.crossplane.io/install-name"
```

Comment:
These looks quite similar what is trying to be introduced here: https://github.com/crossplaneio/crossplane/pull/961/files#diff-4daf64a45270834c13f7f119f2d6aa98R82-R84
Would it make sense to use same labels?

Reply:
Unsure - seems likely though.

Reply:
I like the very specific labels that call out that the 'stackinstall' parent is the named resource, but I think that the abstract labels provided in #961 work just as well and can be used for general tracing processes.

#961 does not include a label for the UID, so I wonder if that should be a requirement.

Reply:
that all makes a lot of sense to me.  it would be nice for this effort to go ahead and use the labels that we intend to use elsewhere for parent/child relationships in #961.  I'll continue driving this effort here to completion.